### PR TITLE
feat: add conops

### PR DESCRIPTION
**Pull request by ServiceDog**

### CODE REVIEW

File: docs/conops.md [hunk 1]
```diff
@@ -1,3 +1,178 @@
...
+Maintainer(s) | @arrow-air/services
+Stuckee | A.M. Smith ([@ServiceDog](https://github.com/ServiceDog))
+
+:exclamation: This document is intended for internal use.
+
+## 1.0 Project Description
+
+#### 1.0.1 Background
+
+This service provides an interface for customers to schedule and manage cargo shipments.
```
Comment (GoodluckH) [inline docs/conops.md:20]: What do you mean by "manage"?  Are you expecting additional features that allow users to manage existing shipments like tracking or cancel or reroute?

File: docs/conops.md [hunk 2]
```diff
@@ -1,3 +1,178 @@
...
+| [High-Level Concept of Operations (CONOPS)](https://github.com/Arrow-air/se-services/blob/develop/docs/conops.md) | Overview of Arrow microservices.                             |
+| [High-Level Interface Control Document (ICD)](https://github.com/Arrow-air/se-services/blob/develop/docs/icd.md)  | Interfaces and frameworks common to all Arrow microservices. |
+[Requirements - `svc-cargo`](https://docs.google.com/spreadsheets/d/1OliSp9BDvMuVvGmSRh1z_Z58QtjlSknLxGVdVZs2l7A/edit#gid=0) | Requirements and user stories for this microservice.
+[Software Design Document - `svc-cargo`](./sdd.md) | Defines the implementation of this interface.
+[Interface Control Document (ICD) - `svc-cargo`](./icd.md) | Defines the inputs and outputs of this microservice.
+
+## 3.0 Needs, Goals and Objectives of Envisioned System
+
+This service is expected to:
+- Handle many (100+) client requests per second for a region of operation.
```
Comment (GoodluckH) [inline docs/conops.md:83]: Have we tested this? I think the performance burden is more on the storage side to handle concurrent read/write ops.

Comment (ServiceDog) [reply docs/conops.md:83]: This is long term objectives

File: docs/conops.md (hunk 1 above)
Comment (ServiceDog) [reply docs/conops.md:20]: Modify (departure or destination ports, times), cancel, etc.

We already support cancellation

Comment (ServiceDog) [reply docs/conops.md:20]: Changed "manage" to "modify"

File: docs/conops.md (hunk 2 above)
Comment (ServiceDog) [reply docs/conops.md:83]: I can remove the (100+) bit, figured "many" doesn't mean anything without context

File: docs/conops.md (hunk 1 above)
Comment (GoodluckH) [reply docs/conops.md]: Probably change it to "This service provides an interface for shippers to book, schedule and modify open shipments."?

These are flight plans have not picked up by operators.

Comment (ServiceDog) [reply docs/conops.md]: Updated to "This service provides an interface for shippers to book, modify, and cancel cargo shipments."

**Review by GoodluckH — APPROVED** [review]

**Pull request merged**